### PR TITLE
Shashank dev

### PR DESCRIPTION
**Pull request by shashankpandey56**
All Assignments

### CODE REVIEW

**Comment by veer-auro** [comment]
@shashankpandey56 Can you re-upload assignment 1 in text format? It's technically supposed to me a markdown file
For flow charts you can screenshot and upload if you are having an issue

File: Week-1/index.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            <th>4.Work From Home</th>
+            <th>Cat Harmony feat Colla</th>
+            <th>3:34</th>
+            <th><a href="#">CATY CLOUD SYNC</a></th>
+          </tr>
+        </table>
+      </div>
+      <div class="third-ele">
+        <div class="image2">
+          <img src="./cattyboard.jpg" width="150" height="150" class="center" />
```
Comment (veer-auro) [inline Week-1/index.html:78]: alt tag

File: Week-1/index.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        </div>
+        <div class="third-content">
+          <h3 style="color: white">If It Ain't Love</h3>
+          <p style="margin-top: -10px; color: white">Catson Derulo</p>
+        </div>
+        <div class="buttons-3">
+          <span style="padding-left: 8%; padding-right: 8%"
```
Comment (veer-auro) [inline Week-1/index.html:85]: move to style.css

File: Week-1/index.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+          </tr>
+        </table>
+      </div>
+      <div class="third-ele">
+        <div class="image2">
+          <img src="./cattyboard.jpg" width="150" height="150" class="center" />
+        </div>
+        <div class="third-content">
+          <h3 style="color: white">If It Ain't Love</h3>
+          <p style="margin-top: -10px; color: white">Catson Derulo</p>
```
Comment (veer-auro) [inline Week-1/index.html:82]: move all inline to style.css

File: Week-1/index.html [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+      <a href="#"><i class="fa fa-fw fa-home"></i></a>
+      <a href="#"><i class="fa fa-fw fa-search"></i></a>
+      <a href="#"><i class="fa fa-fw fa-volume-up"></i></a>
+      <a href="#"><i class="fa fa-fw fa-user"></i></a>
+      <a href="#"><i class="fa fa-fw fa-spotify"></i></a>
+      <a href="#"><i class="fa fa-fw fa-cog"></i></a>
+      <a href="#"><i class="fa fa-fw fa-soundcloud"></i></a>
+    </div>
+    <div class="content">
+      <div class="first-ele">
```
Comment (veer-auro) [inline Week-1/index.html:25]: is this class name typo?

File: Week-1/style.css [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+  float: left;
+  margin-left: -75px;
+  margin-top: 30px;
+}
+.buttons-3 {
+  margin-top: 72px;
+  color: white;
+  width: 60%;
+  margin-top: 150px;
+
```
Comment (veer-auro) [inline Week-1/style.css:96]: remove space

**Review by veer-auro — CHANGES_REQUESTED** [review]
Some comments. 
Also the bottom section is not taking bottom of screen. Instead is visible on scrolling. Everything should be visible without scrolling. 
Screenshot 2022-11-15 at 1 42 03 PM


``

**Comment by veer-auro** [comment]
Also you are supposed to use line awesome icons as per guidelines. Just check if you could replace them. Implementation should be similar.

File: Week-1/index.html (hunk 4 above)
Comment (shashankpandey56) [reply Week-1/index.html:25]: no this is not typo

File: Week-1/index.html (hunk 1 above)
Comment (shashankpandey56) [reply Week-1/index.html:78]: in my screen it is not scrolling after viewport

**Review by veer-auro — APPROVED** [review]

**Review by veer-auro — APPROVED** [review]
LGTM

**Pull request merged**